styles/modules/_search.scss

```diff
@@ -179,13 +179,6 @@
       padding: 0.3em 0px;
     }
 
-    input[type="radio"] + label:before {
```

Comment:
This earlier fix isn't needed any more now that the radio button styling is cleaned up and unified.